### DISCUSSION TITLE
Sample JDL /jh for Dot net core gateway and mircoservices

### DISCUSSION THREAD

**Discussion by nizametech** [jhipster/jhipster-dotnetcore#932] [Q&A]
Could you have sample of jdl /jh  for Dot Net core with gateway, microservices and entities, with various services which are supported.

**Comment by ivanmonteiro** [comment]
Like: https://github.com/jhipster/jdl-samples/blob/main/microservices-blog-store.jh and https://github.com/jhipster/jdl-samples/blob/main/microservice-ecommerce-store.jdl?
